cmd/operator/main.go

```diff
@@ -57,6 +57,8 @@ func main() {
 		WatchdogCanPatchPodLabels:           opts.WatchdogCanPatchPodLabels,
 		APIPort:                             opts.APIPort,
 		HealthzPort:                         opts.HealthzPort,
+		ActorsServiceName:                   opts.ActorsServiceName,
```

Comment:
How about we pass a more generic "additional-control-plane-services" flag? Which can use `[]string` (we use pflag now so we can use `[]string` and it supports repeated values or CSV)

This way it's more generic for the future

PS: We should probably add the same flag for Sentry so it can issue certs without hardcoding the control plane services names

Reply:
> How about we pass a more generic "additional-control-plane-services" flag? Which can use `[]string` (we use pflag now so we can use `[]string` and it supports repeated values or CSV)
> 
> This way it's more generic for the future
> 
> PS: We should probably add the same flag for Sentry so it can issue certs without hardcoding the control plane services names

I have incorporated changes